src/lib/NotionSDK.ts

```diff
@@ -567,6 +568,7 @@ export class NotionSDK implements INotionSDK {
         try {
             const { name, parent } = page;
             const { title } = prop;
+            const nameWithoutEmoji = name.replace("📄", "");
```

Comment:
we can avoid long variable name here.

```suggestion
            const { parent } = page;
            const { title } = prop;
            const name = page.name.replace("📄", "");
```

Reply:
Oki